alacritty.yml

```diff
@@ -353,6 +358,9 @@
   # See `cursor.style` for available options.
   #vi_mode_style: None
 
+  # Cursor blinking interval in milliseconds.
+  #blink_interval: 500
```

Comment:
I think 700 or 650 should be a bit better for default, since they are no that annoying like 500.